.github/workflows/_build_iwasm_release.yml

```diff
@@ -0,0 +1,89 @@
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+name: build iwasm release
```

Comment:
Must the file name begin with `_`?

Reply:
just a habit. not necessary. Any suggestion?

Reply:
How about removing the prefix `_` for these files? And rename `fetch_compare_version.py` to `fetch_and_compare_version.py`?